Makefile

```diff
@@ -10,43 +10,50 @@ MOCKERY_BIN=$(shell which mockery || "./bin/mockery")
 # aws-sdk-go/private/model/api package is gated behind a build tag "codegen"...
 GO_TAGS=-tags codegen
 
-.PHONY: all build-ack-generate test clean-mocks mocks build-controller build-kind-cluster build-kind-cluster-preserve \
+.PHONY: all build-ack-generate test clean-mocks mocks build-controller-image build-controller build-kind-cluster build-kind-cluster-preserve \
         build-kind-cluster-functional kind-get-cluster-names delete-all-kind-clusters
 
 all: test
 
-build-ack-generate:
+build-ack-generate:	## Build ack-generate binary
 	go build ${GO_TAGS} -o bin/ack-generate cmd/ack-generate/main.go
 
-test: | mocks
+test: | mocks	## Run code tests
 	go test ${GO_TAGS} ./...
 
-clean-mocks:
+clean-mocks:	## Remove mocks directory
 	rm -rf mocks
 
-build-controller:
+build-controller-image: build-controller	## Build container image for SERVICE
```

Comment:
Let’s remove build-controller action from this step. This will give false information if build-controller-image step fails.